### DISCUSSION TITLE
make error

### DISCUSSION THREAD

**Discussion by jkheo47** [libMesh/libmesh#3775] [Q&A] [answered]
../LIBMESH/src/solvers/petsc_linear_solver.C:1570:10: error: duplicate case value: 'KSP_CONVERGED_CG_CONSTRAINED' and 'KSP_CONVERGED_STEP_LENGTH' both equal '6'
    case KSP_CONVERGED_STEP_LENGTH     : return CONVERGED_STEP_LENGTH;

Log file is here:    
[config.log](https://github.com/libMesh/libmesh/files/14114327/config.log)
[make.log](https://github.com/libMesh/libmesh/files/14114328/make.log)

How to fix it?

**Comment by jwpeterson** [comment, marked as answer]
Sounds like the same issue as in #3767? In brief, you cannot use the latest PETSc with the libMesh release (1.7.1). You'll either need to use libmesh master or an older PETSc to get things working.

**Reply by jkheo47** [reply]
Thanks for your help.
